frida_tools/tracer.py

```diff
@@ -573,6 +603,21 @@ def stop(self):
             self._script = None
 
     def _create_trace_script(self):
+        # Extract all init session files into a single string,
+        # substitute newlines with '\u000a'
```

Comment:
As outlined in the [style guide](https://github.com/frida/frida/blob/master/CONTRIBUTING.md), this comment should probably be omitted in favor of naming variables, or a helper function named appropriately.

Reply:
I hadn't noticed this style guide, although you pointed several issues out to me.  Thanks for the pointer!

Reply:
I have removed the comments from the beginning of the function, but feel it is crucial to document the two lines that insert '\u000a':

```
def _create_trace_script(self):
    self._init_session_code = ''
    for path in self._init_session_paths:
        # Append contents of <path> to <self._init_session_code>
        with open(path, 'rt') as fo
            line = fo.readlines()

        for i in range (len(line)):
            line [i] = line [i].rstrip ()

        # eval() does not like 0x0A, replace with '\u000a'
        self._init_session_code += '\\u000a'.join(line)

        # Insert '\ua000' in case another init session follows
        self._init_session_code += '\\u000a'

    return """'use strict';
```
It is too easy for another maintainer to restore the '\n' sequence, which will break the entire feature.  What do you think?

Reply:
Yes definitely want to explain this bit. The `# Append contents` comment above can be removed though.

Reply:
The `#Append contents` should of course be deleted (don't know what I was smoking when I left it in :-))